csv-core/src/writer.rs

```diff
@@ -119,6 +127,17 @@ impl WriterBuilder {
         self.wtr.double_quote = yes;
         self
     }
+
+    /// The comment character that will be used when later reading the file.
+    ///
+    /// If `quote_style` is set to `QuoteStyle::Necessary`, a field will
+    /// be quoted if the comment character is detected anywhere in the field.
+    ///
+    /// The default value is None.
+    pub fn comment(&mut self, comment: Option<u8>) -> &mut WriterBuilder {
+        self.wtr.comment = comment;
+        self
+    }
```

Comment:
Please add a test for this new option in the tests at the bottom of this file.

Reply:
There should be a test at the bottom of _this_ file, which is in `csv-core`, not `csv`.